config/Mosaic.py

```diff
@@ -1,3 +1,8 @@
 import os
 from lsst.utils import getPackageDir
 config.astrom.load(os.path.join(getPackageDir("obs_subaru"), "config", "filterMap.py"))
+
+from lsst.pipe.tasks.setConfigFromEups import setPhotocalConfigFromEups
+setPhotocalConfigFromEups(config)
+
+config.load(os.path.join(getPackageDir("obs_subaru"), "config", "srcSchemaMap.py"))
```

Comment:
Do we need both `config/Mosaic.py` and `config/hsc/Mosaic.py`? As far as I know, you've only tested on HSC; I'm going to go out on a limb and speculate that meas_mosaic as-is will break if it's applied to SuprimeCam.

Assuming we do need them both, should `srcSchemaMap` live at this level or in `hsc`?